examples/twitter-nlp/requirements.txt

```diff
@@ -0,0 +1,4 @@
+click==8.0.1
+colorama==0.4.4
+deepsparse>=0.11
+git+https://github.com/twintproject/twint@master#egg=twint
```

Comment:
is there a python package for this? pointing to the master branch of a repo doesn't seem great for QA stability

Reply:
there is, but the current python package is broken. Only the current main works -- I'll update it to point out the current commit on main